docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -163,3 +164,12 @@ In this case, try increasing the memory for the {kib} instance to 2GB.
 === ARM systems
 
 Chromium is not compatible with ARM RHEL/CentOS.
+
+[float]
+[[reporting-troubleshooting-maps-ems]]
+=== Unable to connect to Elastic Maps Service
+
+https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
+tile layers and vector shapes of administrative boundaries.
+Reports containing Maps with missing basemap layers or administrative boundaries is due to the kibana server not having access to EMS.
```

Comment:
If Maps refers to the app, then ok as uppercase.  Otherwise, use "maps".

Does this work?

If a report contains a map with a missing basemap layer or administrative boundary, the {kib} server does not have access to EMS.